Makefile

```diff
@@ -0,0 +1,36 @@
+.PHONY: all root-ssl server-ssl client-ssl
```

Comment:
Can you move this to the certs dir and add a comment explaining what it does?

Reply:
done 7da16d9